overlord/configstate/configcore/corecfg.go

```diff
@@ -35,12 +33,10 @@ var (
 )
 
 type Conf interface {
-	Get(snapName, key string, result interface{}) error
-	Set(snapName, key string, value interface{}) error
-	Changes() []string
-	State() *state.State
+	config.Conf
```

Comment:
when everything else is agreed can we avoid this "wrapping", I understand it avoids churn right now, but going forward is a bit strange

Reply:
Yes, with pleasure.

Reply:
to be clear this bit can be in a small follow-up

Reply:
I ended up doing it just now.